Tests/MintTests/MintTests.swift

```diff
@@ -68,20 +68,23 @@ class MintTests: XCTestCase {
 
         // check that not globally installed
         XCTAssertFalse(globalPath.exists)
+        XCTAssertEqual(mint.getGlobalInstalledPackages(), [:])
 
         // install already installed version globally
         try mint.install(repo: testRepo, version: testVersion, command: testCommand, global: true)
 
         XCTAssertTrue(globalPath.exists)
         let globalOutput = main.run(globalPath.string)
         XCTAssertEqual(globalOutput.stdout, testVersion)
+        XCTAssertEqual(mint.getGlobalInstalledPackages(), [testCommand: testVersion])
 
         // install latest version
         let latestPackage = try mint.install(repo: testRepo, version: "", command: testCommand, global: true)
         expectMintVersion(package: latestPackage)
         XCTAssertEqual(latestPackage.version, latestVersion)
         let latestGlobalOutput = main.run(globalPath.string)
         XCTAssertEqual(latestGlobalOutput.stdout, latestVersion)
+        XCTAssertEqual(mint.getGlobalInstalledPackages(), [testCommand: latestVersion])
```

Comment:
Can we also check that `getGlobalInstalledPackages()` returns an empty dictionary after we uninstall down below